### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by nstepien**

### CODE REVIEW

File: test/column/cellClass.test.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+test('cellClass is undefined', () => {
+  const columns: readonly Column<Row>[] = [{
+    key: 'id',
+    name: 'ID',
+    cellClass: undefined
+  }];
+  setup({ columns, rows });
+  expect(getAllCellClassNames()).toStrictEqual([
```
Comment (amanmahajan7) [inline test/column/cellClass.test.ts:22]: Wdyt about using https://github.com/testing-library/jest-dom#tohaveclass?

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-assertion

Comment (nstepien) [reply test/column/cellClass.test.ts:22]: Sounds good. I've had this blog post bookmarked for a while now, I should get to it at some point.

Comment (nstepien) [reply test/column/cellClass.test.ts:22]: image
The error messages can be buggy 🤔

**Comment by nstepien** [comment]
Not a fan that `toHaveTextContent` does not have an `exact` option like `toHaveClass`.
https://github.com/testing-library/jest-dom#tohavetextcontent

**Review by amanmahajan7 — APPROVED** [review]

**Pull request merged**